src/server/routes/__tests__/licences.integration.spec.js

```diff
@@ -0,0 +1,93 @@
+import { contactForLicensee } from '@defra-fish/dynamics-lib'
+import initialiseServer from '../../server.js'
+
+const mockResponse = {
+  ContactId: 'contact-identifier-111',
+  FirstName: 'Fester',
+  LastName: 'Tester',
+  DateOfBirth: '9/13/1946 12:00:00 AM',
+  Premises: '47',
+  Street: null,
+  Town: 'Testerton',
+  Locality: null,
+  Postcode: 'WA4 1HT',
+  ReturnStatus: 'success',
+  SuccessMessage: 'contact found successfully',
+  ErrorMessage: null,
+  ReturnPermissionNumber: '11100420-2WT1SFT-B7A111',
+  oDataContext:
+    'https://api.com/api/data/v9.1/$metadata#Microsoft.Dynamics.CRM.defra_GetContactByLicenceAndPostcodeResponse'
+}
+
+const noContactResponse = {
+  ContactId: null,
+  FirstName: null,
+  LastName: null,
+  DateOfBirth: null,
+  Premises: null,
+  Street: null,
+  Town: null,
+  Locality: null,
+  Postcode: null,
+  ReturnStatus: 'error',
+  SuccessMessage: '',
+  ErrorMessage: 'contact does not exists',
+  ReturnPermissionNumber: null,
+  oDataContext:
+    'https://api.crm4.dynamics.com/api/data/v9.1/$metadata#Microsoft.Dynamics.CRM.defra_GetContactByLicenceAndPostcodeResponse'
+}
+
+describe('licences', () => {
+  /** @type {import('@hapi/hapi').Server} */
+  let server = null
+
+  beforeAll(async () => {
+    server = await initialiseServer({ port: null })
+  })
+
+  afterAll(async () => {
+    await server.stop()
+  })
+
+  describe('GET /api/licence', () => {
+    it('should return 200 if licence number and postcode are valid', async () => {
```

Comment:
tiny point, but you're checking the format of the response payload rather than the response code